k3os/overlay/libexec/k3os/mode

```diff
@@ -44,8 +44,11 @@ fi
 done
 
 if [ -z "$MODE" ]; then
-    pfatal "Failed to determine boot mode (did you forget to set k3os.mode?)"
-elif [ ! -e $SCRIPTS/mode-${MODE} ]; then
+  echo "Failed to determine boot mode. Fallback to install mode."
```

Comment:
Can we set FALLBACK_MODE to "install"(before the mode-wait loop) if it is not specified?

Reply:
Won't this make the loop only run the first iteration? It's possible for a disk that contains the `HARVESTER_STATE` label to appear later. We might want the loop to run 30 times to make sure there is no disk that contains the label.